src/client/input/sdl.c

```diff
@@ -130,9 +131,38 @@ static cvar_t *joy_axis_triggerright_threshold;
 // Joystick haptic
 static cvar_t *joy_haptic_magnitude;
 
+// Gyro mode (0=off, 3=on, 1-2=uses button to enable/disable)
+cvar_t *gyro_mode;
+cvar_t *gyro_turning_axis;	// yaw or roll
+
+// Gyro sensitivity
+cvar_t *gyro_yawsensitivity;
+cvar_t *gyro_pitchsensitivity;
+
+// Gyro availability
+qboolean gyro_hardware = false;
+
+// Gyro is being used in this very moment
+static qboolean gyro_active = false;
+
+// Gyro calibration
+static qboolean calibrating_gyro = false;
+static float gyro_accum[3];
+static unsigned int num_samples;
+
+static cvar_t *gyro_calibration_x;
+static cvar_t *gyro_calibration_y;
+static cvar_t *gyro_calibration_z;
+
 // Support for hot plugging of game controller
 static qboolean first_init = true;
-static int init_delay = 30;
+static int in_delay = 30;
```

Comment:
Could we use old variable naming?

Reply:
Yes, old name is no longer representative of what this counter is doing now. I invented that variable on an old PR precisely to add a "delay" to gamepad initialization on hotplug (since adding them inmediately after detected led to unwanted behaviour). On the new PR, as this variable existed, I didn't want to add another "counter" for the amount of inputs needed in the "gyro calibration" phase, so I reused it and wanted to reflect that in the name, with "in_" as "input", like in IN_Init(), IN_Update(), etc.

Still, I agree the name is ugly and doesn't clarify what it's doing now, but at least expresses it's more of a "general purpose" variable.

Of course, I'll change the name; I just don't think the old one explains both uses. Is "updates_counter" appropiate for the project? Any suggestions are welcome.